eslint.config.ts

```diff
@@ -82,6 +94,16 @@ export default defineConfig(
 	{
 		extends: [packageJson.configs["recommended-publishable"]],
 		files: ["package.json"],
+		plugins: {
+			// @ts-expect-error types mismatch
```

Comment:
Can you link to a tracking issue? (if one doesn't exist, let's file one)

Reply:
Done